server/project.go

```diff
@@ -25,6 +25,9 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 		if err := jsonapi.UnmarshalPayload(c.Request().Body, projectCreate); err != nil {
 			return echo.NewHTTPError(http.StatusBadRequest, "Malformatted create project request").SetInternal(err)
 		}
+		if projectCreate.TenantMode == api.TenantModeTenant && !s.feature(api.FeatureMultiTenancy) {
```

Comment:
@tianzhou shall we allow users to create tenant mode project to get a sense but not allowing them to operate (create db, update schema, transfer db, etc) on it?

Reply:
I think the current gating point is appropriate. If we let customer create the tenant mode project, add many databases and then tell them to upgrade in order to use the feature. I am afraid that goes too much.

Reply:
SG